op-service/testutils/devnet/anvil.go

```diff
@@ -32,41 +26,74 @@ type Runner struct {
 	port      int32
 }
 
-func New(l1RPCURL string, logger log.Logger) (*Runner, error) {
-	return NewWithOpts(l1RPCURL, "1000000000", logger)
+type AnvilOption func(*Anvil)
+
+func WithForkURL(url string) AnvilOption {
+	return func(a *Anvil) {
+		a.args["--fork-url"] = url
+	}
+}
+
+func WithBaseFee(baseFee uint64) AnvilOption {
+	return func(a *Anvil) {
+		a.args["--base-fee"] = strconv.FormatUint(baseFee, 10)
+	}
+}
+
+func WithBlockTime(bt int) AnvilOption {
+	if bt < 0 {
+		panic("block time must be non-negative")
+	}
+	return func(a *Anvil) {
+		a.args["--block-time"] = strconv.Itoa(bt)
+	}
+}
+
+func WithChainID(id uint64) AnvilOption {
+	return func(a *Anvil) {
+		a.args["--chain-id"] = strconv.FormatUint(id, 10)
+	}
 }
 
-func NewWithOpts(l1RPCURL string, baseFee string, logger log.Logger) (*Runner, error) {
+func NewAnvil(logger log.Logger, opts ...AnvilOption) (*Anvil, error) {
 	if _, err := exec.LookPath("anvil"); err != nil {
 		return nil, fmt.Errorf("anvil not found in PATH: %w", err)
 	}
 
-	args := []string{"--port", "0", "--base-fee", baseFee}
-	if l1RPCURL != "" {
-		args = append([]string{"--fork-url", l1RPCURL}, args...)
+	a := &Anvil{
+		args: map[string]string{
+			"--base-fee": "1000000000",
+			"--port":     "0",
+		},
```

Comment:
Just confirming only, are these defaults enough?

Reply:
yep - these were the previous defaults. port needs to be 0 so we can avoid port conflicts (it uses an OS assigned port) and the base fee is a sensible default.